ui/app/adapters/sync/association.js

```diff
@@ -20,19 +20,20 @@ export default class SyncAssociationAdapter extends ApplicationAdapter {
     return `${super.buildURL()}/destinations/${destinationType}/${destinationName}/associations${uri}`;
   }
 
-  query(store, { modelName }, query) {
-    // endpoint doesn't accept the typical list query param and we don't want to pass options from lazyPaginatedQuery
```

Comment:
need to circle back to this about "passing options from lazyPaginatedQuery"

alternative is just piping the url in directly below instead of calling `this.query`